### DISCUSSION TITLE
Make selection super seed

### DISCUSSION THREAD

**Discussion by pete2583** [Novik/ruTorrent#2276] [Ideas]
Hello, 

I wondered if it is possible to me able to select a few torrents and have an option to make all selected torrents superseed from the right click menu. It is possible in qBittorrent and it's about the best thing in qBittorrent if I'm honest.

Cheers.

**Comment by stickz** [comment]
Yes this is possible. It's a great idea. I looked on the rTorrent wiki, we just need to set `d.connection_current.set=initial_seed` for the selected torrents. https://github.com/rakshasa/rtorrent/wiki/Using-initial-seeding
